gtsfm/visualization/view_scene_olsson.py

```diff
@@ -0,0 +1,78 @@
+"""
+Script to render Olsson dataset ground truth camera poses and structure, and render the scene to the GUI.
+
+Results must be stored in Carl Olsson's `data.mat` file format.
+
+Authors: John Lambert
+"""
+import argparse
+import os
+from pathlib import Path
+
+import numpy as np
+
+import gtsfm.visualization.open3d_vis_utils as open3d_vis_utils
+from gtsfm.loader.olsson_loader import OlssonLoader
+
+
+TEST_DATA_ROOT = Path(__file__).resolve().parent.parent.parent / "tests" / "data"
+
+# The Olsson loader requires we set a value for the frame lookahead, but it will not affect the visualization.
+DUMMY_MAX_FRAME_LOOKAHEAD = 1
+
+
+def view_scene(args: argparse.Namespace) -> None:
```

Comment:
Maybe we could make this into a more general module in a future PR? Since this just called the respective dataloader to read in the GT point cloud, maybe we don't need a separate file for each dataset?